File: src/main/java/com/dashboard/models/JPA/Message.java
package com.dashboard.models.JPA;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.Date;

@Data
@Entity
@Table(name = "message")
@NoArgsConstructor
public class Message {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;

    @Column
    private String message;

    @Column
    @Temporal(TemporalType.DATE)
    private Date created;

    public Message(String message){
        this.message = message;
    }
}

File: src/main/kotlin/models/jpa/Message.kt
package models.jpa

import java.util.*
import javax.persistence.*

@Entity
@Table(name = "message")
data class Message(
    @Column val message: String,
    @Id @GeneratedValue(strategy = GenerationType.AUTO) val id: Int,
    @Temporal(TemporalType.DATE) val created: Date)